pages/linux/nsxiv.md

```diff
@@ -0,0 +1,32 @@
+# nsxiv
+
+> Neo Simple X Image Viewer.
+> More information: <https://nsxiv.codeberg.page/man>.
+
+- Open images:
+
+`nsxiv {{path/to/file1 path/to/file2 ...}}`
+
+- Open images from directories in image mode:
+
+`nsxiv {{path/to/directory1 path/to/directory2 ...}}`
+
+- Search directories recursively for images to view:
+
+`nsxiv -r {{path/to/directory1 path/to/directory2 ...}}`
+
+- Quit nsxiv:
+
+`q`
+
+- Switch to thumbnail mode / open selected image in image mode:
+
+`Return`
+
+- Count images forward in Image mode:
+
+`n, Space`
```

Comment:
```suggestion
`Space + N`
```

Reply:
User is supposed to either press `space` or `n`. `Space + n` does not seem suitable. Also, I've written it the way it was in the man pages. 

Why does it have to be `N` instead of `n`.

If this is the standard, then I'll go ahead and make these changes. Btw, I took a look at vim's tldr and wrote these.

Reply:
> User is supposed to either press `space` or `n`. `Space + n` does not seem suitable. Also, I've written it the way it was in the man pages.
> 
> Why does it have to be `N` instead of `n`.
> 
> If this is the standard, then I'll go ahead and make these changes. Btw, I took a look at vim's tldr and wrote these.

I agree with you, `Space + N` would confuse users as they might think it is a single command/keyboard shortcut. Also, I noticed the page had separate flags for both lower and upper case letters. Let's see what @CleanMachine1 thinks.

Reply:
We should just pick either `n` or `space` then, I think the most common in your opinion would be best.

Reply:
Keeping `n` & `p`